go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/utsavanand2/k3sup
```

Comment:
Please can you remove the Go mod file and reinstate the Gopkg.toml? I mentioned on Slack that `dep` is used and there's a link on the README.md saying to use the contribution guide from OpenFaaS (OpenFaaS uses dep currently)